internal/upload/koji/koji.go

```diff
@@ -420,3 +424,24 @@ func GSSAPICredentialsFromEnv() (*GSSAPICredentials, error) {
 		KeyTab:    keyTab,
 	}, nil
 }
+
+func CreateKojiTransport(relaxTimeout uint) *http.Transport {
```

Comment:
Should we just fold this into `kojiNew()`? We now take a generic `http.Transport` there, but if the comments in this function is to be believed we can't really pass anything, maybe just hard-code the transport setup?

I think @ondrejbudai did this originally, so might have thoughts?

Reply:
I think I wanted to make the `koji` library more flexible but I guess it isn't actually needed so I have no problems with just folding the code in.